Add link to event correlation public beta signup in public doc (#19076)
DataDog/documentation: content/en/service_management/events/event_management/_index.md

---
title: Event Management
kind: documentation
description: Learn how to use Event Management in your pipeline.
is_beta: true
---

{{< site-region region="gov" >}}
<div class="alert alert-warning">Event Management is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
{{< /site-region >}}

{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeYkh0jFy_wMCLGKZ5019H0DpFvq0fILvyJEt_gRyeGgvRymA/viewform" header="false" >}}

> Event Management are in private beta. To request access, complete the form.

Reviewer: ```suggestion
Event Management is in private beta. To request access, complete the form.
```

{{< /callout >}}

> Event Management allows you to aggregates events from a company IT infrastructure or services to reduce alert noise, proactively surface undetected issues, and trigger incidents or auto-remediation actions.

Reviewer: ```suggestion
Event Management allows you to aggregate events from company IT infrastructure or services to reduce alert noise, proactively surface undetected issues, and trigger incidents or auto-remediation actions.
```

## Set up Event Management
There are two main ways to correlate events:

> 1. Rule based. 
> 2. Topology based. 

Reviewer: ```suggestion
1. Rule based
2. Topology based
```

## Rule based

> To create Event Correlation rules for your organization.

Reviewer: ```suggestion
To create Event Correlation rules for your organization:
```

1. In Datadog, navigate to [**Service Management** > **Event Management** > **Correlation Rules**][1] and click **+ Add a Correlation Rule**.

> 2. Select the source of the events. You can choose alerting source like monitor alert and non-alerting source like kubernetes. 

Reviewer: ```suggestion
2. Select the source of the events. You can choose an alerting source, like a monitor alert, or a non-alerting source, like Kubernetes. 
```
